Resources List

hug33k opened the pull request:
Added a new section to Trackit : Resources List

This section let you see more details for your resources such as EC2, RDS, ...

## Features

- Section "Resources List" is available in the Navigation Bar
- EC2
  - A table is available to see data. It will show you basic informations (Name, ID, KeyPair, Region, Type, ...) but also usage stats (CPU, Network, I/O) and tags.
  - You can use filters (Name, ID, KeyPair, Region, Type) to search for a specific instance.

## Notes

- A PopOver will be added to display more details on I/O usage (More detailed list for internal volume & EBS)
- A column "Info" will be added to display more details for each instance.
- A column "Status" will be added to display a badge based on the instance status (Running, stoped, ...)

alexis-ld commented on src/components/aws/resources/VMsComponent.js:
```diff
@@ -0,0 +1,350 @@
+                Header: 'Read',
+                accessor: 'ioRead',
+                filterable: false,
+                Cell: row => formatBytes(Object.keys(row.value).map((volume) => (row.value[volume])).reduce((a, b) => (a+b)))
```
![screen shot 2018-08-13 at 16 49 57](https://user-images.githubusercontent.com/3480526/44039359-3a95d206-9f19-11e8-9b89-4e5e0796a32e.png)

The app is crashing right here for me.

hug33k commented:
@alexis-ld I updated this PR :
- I merged `staging` branch
- I made some small updates on UI for filters fields
- I added support for RDS into Databases section

alexis-ld commented:
Couples other changes : 

1.  The filter inputs look a bit weird with the border-radius and the box-shadow : 
![screen shot 2018-08-21 at 14 01 09](https://user-images.githubusercontent.com/3480526/44400371-2fed2100-a54b-11e8-90c1-4f7bad05c852.png)
Maybe tweak the styling a bit so it's more like that : 
![screen shot 2018-08-21 at 14 02 41](https://user-images.githubusercontent.com/3480526/44400386-3b404c80-a54b-11e8-8f40-4ba9545b2593.png)

2.  Reorder the EC2 columns. You can push back the key pair column at the end as it is less useful than others. Bring the Tags column in the first columns. It will bring more colors without having to scroll horizontally.

3.  Table ordering doesn't work on some columns : 
![screen shot 2018-08-21 at 14 03 07](https://user-images.githubusercontent.com/3480526/44400500-a0943d80-a54b-11e8-87fa-5c3f52efad1a.png)
Please try all columns with both ascending and descending to make sure ordering works well. It's kinda inconsistent. It may come from the fact that you use the formatted version to order, there are some ways to order with non formatted values in react-table.

4.  Databases table lack colors compared to EC2 table
![screen shot 2018-08-21 at 14 09 50](https://user-images.githubusercontent.com/3480526/44400571-ee10aa80-a54b-11e8-9385-b29db16f678d.png)
Just a touch a color could be enough here since we don't have much data. Maybe just replace the "Yes/No" column for Multi-AZ by a small coloured icon or something like that.
